scm/src/GFS_typedefs.F90

```diff
@@ -3290,6 +3537,15 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     Model%fhlwr            = fhlwr
     Model%nsswr            = nint(fhswr/Model%dtp)
     Model%nslwr            = nint(fhlwr/Model%dtp)
+    if (restart) then
```

Comment:
Have you tried this in SCM? What differences do you see? Would be interesting to check (not for merging this PR, though).

Reply:
Nope. Restarts are not implemented in the SCM since they are so short to run. This is another GFS_typedefs variable that is only there to minimize differences between FV3/SCM. There should probably be a list of namelist and other vars to not change from their default in the SCM.